### DISCUSSION TITLE
Column level masking with apache ranger not working

### DISCUSSION THREAD

**Discussion by debimishra89** [trinodb/trino#16459] [Q&A]
I have integrated Trino v388 with Ranger v2.3
I am able to access-control trino tables and columns in postgresql and hive catalogs.
But Not able to do column level masking and row-level filtering
There is no error, It looks like it simply ignores the mask and row- filter policies.
any advice?

**Comment by atifiu** [comment]
Have you tried using file based access control in trino for masking? Also how you have integrated Trino and Ranger, are you using Trino plugin for Ranger ?

**Reply by atifiu** [reply]
@aakashnand Could you please help with column masking and row level filtering.

**Reply by javsanbel2** [reply]
Is there any configuration parameter that needs to be set to enable row level filter policies? I got the same issue, policy is created, resource based policy works, however, row level filter does not.
